docs/Other SDL Features/Uploading Images/index.md

```diff
@@ -8,36 +8,37 @@ You should be aware of these four things when using images in your SDL app:
 
 1. You may be connected to a head unit that does not have the ability to display images.
 2. You must upload images from your mobile device to the head unit before using them in a template.
-3. Persistant images are stored on a head unit between sessions. Ephemeral images are destroyed when a session ends (i.e. when the user turns off their vehicle).
+3. Persistent images are stored on a head unit between sessions. Ephemeral images are destroyed when a session ends (i.e. when the user turns off their vehicle).
 4. Images can not be uploaded when the app's `hmiLevel` is `NONE`. For more information about permissions, please review [Understanding Permissions](Getting Started/Understanding Permissions).
 
 To learn how to use images once they are uploaded, please see [Text, Images, and Buttons](Displaying a User Interface/Text Images and Buttons).
 
 ## Checking if Graphics are Supported
-Before uploading images to a head unit you should first check if the head unit supports graphics. If not, you should avoid uploading unneccessary image data. To check if graphics are supported look at the `SDLManager`'s `registerResponse` property once the `SDLManager` has started successfully.
+Before uploading images to a head unit you should first check if the head unit supports graphics. If not, you should avoid uploading unnecessary image data. To check if graphics are supported, @![iOS]look at the `SDLManager`'s `registerResponse`'s ,`displayCapabilities` property once the `SDLManager` has started successfully.!@ @![android,javaSE,javaEE] use the `getCapability()` method of a valid `SystemCapabilityManager` obtained from `sdlManager.getSystemCapabilityManager()` to find out the display capabilities of the head unit.!@
 
+@![iOS]
 ##### Objective-C
 ```objc
 __weak typeof (self) weakSelf = self;
 [self.sdlManager startWithReadyHandler:^(BOOL success, NSError * _Nullable error) {
     if (!success) {
-        NSLog(@"SDL errored starting up: %@", error);
+        NSLog(@"SDL encountered an error starting up: %@", error);
         return;
-    } 
+    }
 
     SDLDisplayCapabilities *displayCapabilities = weakSelf.sdlManager.registerResponse.displayCapabilities;
```

Comment:
This should be updated to reference the `systemCapabilityManager`

Reply:
fixed